### PR TITLE
Add new website announcement  post

### PR DESCRIPTION
**Pull request by arturi**
Please suggestions in place or just commit your edits, no comments! :-)

### CODE REVIEW

File: blog/2023-05-new-website.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Hexo got less maintained, our docs more messy, and quite a few
+places were collecting dust.
+
+About a year ago Merlijn set on a task to refresh the Uppy documentation
+experience, rewriting most of the docs from the ground up, with everyone on the
+team contributing in their area of Uppy expertise.
+[Docusaurus](https://docusaurus.io/) 🦕 was chosen as one of the top modern tools for
+documentation.
+
+![](/img/blog/new-website/new-uppy-io-2.jpg)
```
Comment (aduh95) [inline blog/2023-05-new-website.md:26]:
```suggestion
![Screenshot of the Uppy website showing a documentation page with new design](/img/blog/new-website/new-uppy-io-2.jpg)
```

File: blog/2023-05-new-website.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Most of the plugin’s docs now explain when you should use it (and sometimes when
+not), followed by how to install, tips and details and links to CodeSandbox
+samples.
+
+And for desert, we now have a swift website-wide search in the top bar, powered by [Algolia](https://www.algolia.com), give it a go!
+
+
+![](/img/blog/new-website/new-uppy-io-3.jpg)
```
Comment (aduh95) [inline blog/2023-05-new-website.md:35]:
```suggestion
![Screenshot of the Uppy website showing the search interface](/img/blog/new-website/new-uppy-io-3.jpg)
```

File: blog/2023-05-new-website.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+title: 'New Uppy.io and docs'
+date: 2023-05-10
+authors: [murderlon, arturi, aduh95]
+image: 'https://uppy.io/img/blog/new-website/new-uppy-io-1.jpg'
+published: true
+---
+
+<img src="/img/blog/new-website/new-uppy-io-1.jpg" className="border" />
```
Comment (aduh95) [inline blog/2023-05-new-website.md:9]:
```suggestion
<img src="/img/blog/new-website/new-uppy-io-1.jpg" className="border" alt="Screenshot of the new Uppy website homepage" />
```

**Review by Murderlon — APPROVED** [review]
Nice work 👍 

I made some small improvements to the screenshots, like showing more of the new elements on the docs page of the dashboard. All else LGTM

**Review by Murderlon — APPROVED** [review]
Nice work 👍 

I made some small improvements to the screenshots, like showing more of the new elements on the docs page of the dashboard. All else LGTM

**Comment by Murderlon** [comment]
Would be nice to merge #118 before going live though

**Comment by arturi** [comment]
I carefully optimized images to be 50kb in size, your new ones are PNGs of 550kb and 3000px width :scream: Will try to optimize again.

**Comment by Murderlon** [comment]
That's a good point. I forgot to resize and optimise them on https://squoosh.app/ as I usually do.

**Pull request merged**